src/targets/gpu/include/migraphx/gpu/rnn_last_cell_output.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef MIGRAPHX_GUARD_RTGLIB_RNN_LAST_CELL_OUTPUT_HPP
+#define MIGRAPHX_GUARD_RTGLIB_RNN_LAST_CELL_OUTPUT_HPP
+
+#include <migraphx/shape.hpp>
+#include <migraphx/op/rnn_last_cell_output.hpp>
+#include <migraphx/gpu/rnn_last_output.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace gpu {
+
+struct context;
+
+struct hip_lstm_last_cell_output : dev_rnn_last_output<op::lstm_last_cell_output>
+{
+    hip_lstm_last_cell_output() {}
+    hip_lstm_last_cell_output(op::lstm_last_cell_output o) : dev_rnn_last_output(o){};
+};
```

Comment:
Why do we need different operators for this? Its the same operator.

Reply:
Do you mean it is same with hip_rnn_last_hs_output? Yes, the algorithm used is the same, but we need the different name to indicate whether it is the last_hs_output or last_cell_output.

Reply:
You know if its last_hs_output or last_cell_output in rewrite_rnn.

Reply:
fixed